packages/block-editor/src/components/block-list/block.js

```diff
@@ -466,8 +466,15 @@ function BlockListBlock( {
 			aria-label={ blockLabel }
 			childHandledEvents={ [ 'onDragStart', 'onMouseDown' ] }
 			tagName={ animated.div }
-			style={ style }
 			{ ...blockWrapperProps }
+			style={
```

Comment:
This reordering clobbers any styles assigned by `blockWrapperProps`.

Notably: Column widths.

https://github.com/WordPress/gutenberg/blob/6f56a7a704114980902485879a151bb46d7ba3cb/packages/block-library/src/column/index.js#L28-L37

Reply:
What's the difference between `wrapperProps` and `blockWrapperProps` anyways? It's not very clear.

Reply:
I don't understand. I'm merging the styles here 

```
{
						...wrapperProps.style,
						...animationStyle,
					}
```

so it should continue to work?

Reply:
Oh, I guess `blockWrapperProps` just merges into `wrapperProps` anyways:

https://github.com/WordPress/gutenberg/blob/6f56a7a704114980902485879a151bb46d7ba3cb/packages/block-editor/src/components/block-list/block.js#L414-L421

So the solution should probably be that we don't ever reference `wrapperProps` directly after this merge, and instead always use `blockWrapperProps` (either that or we just remove the separate variable and merge into itself).

Reply:
Oh I missed that. yeah, that should work.

Reply:
See #16819